addons/binding/org.openhab.binding.mqtt.generic/README.md

```diff
@@ -323,12 +323,13 @@ The trigger channel will trigger for each received message on the MQTT topic "al
 Now push those changes to your items in a `rules` file:
 
 ```xtend
-rule "Publish all"
+rule "Receive all"
 when 
-      Channel "mqtt:broker:myUnsecureBroker:myTriggerChannel" triggered
-then
-    val parts = receivedEvent.split("#")
-    sendCommand(parts.get(0), parts.get(1))
+      Channel "mqtt:broker:MQTTBroker:myTriggerChannel" triggered
+then 
+    val parts1 = receivedEvent.toString.split("/").get(1)
```

Comment:
Could you please add a comment above this line explaining briefly why a "/" split is required? Thanks

Reply:
I changed two things.
1st I added .toString as the .split method is not valid on the receivedEvent otherwise.
2nd this gives you this String as an output:
'mqtt:broker:myUnsecureBroker:myTriggerChannel triggered allItems/Item#State'
so when split the rule tries to use everything before the # as an Item name.
So by first splitting at the / and only using the second part for the # split you get the correct Item name.
See this log for only using the # split:
```
2019-03-17 19:10:54.251 [vent.ChannelTriggeredEvent] - mqtt:broker:MQTTBroker:myTriggerChannel triggered allItems/Luftfeuchtigkeitinnen#56.1

==> /var/log/openhab2/openhab.log <==

2019-03-17 19:10:54.279 [WARN ] [rthome.model.script.actions.BusEvent] - Item 'mqtt:broker:MQTTBroker:myTriggerChannel triggered allItems/Luftfeuchtigkeitinnen' does not exist.
```

Reply:
or add a comment in the Readme? Sorry these are my first two pull requests

Reply:
Yes, exactly I mean to the readme so that fellow developers can understand what's going on :)